test/fixtures/07-with-cargo-configuration/build.sh

```diff
@@ -0,0 +1 @@
+cargo config -Z unstable-options get target > static/cargo_config.txt
```

Comment:
this looks unrelated to this fixture?

Reply:
This writes what we have defined in `.cargo/config.toml`, so it's intended.